app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -53,20 +53,28 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
     final BuildPushMessage buildPushMessage =
         BuildPushMessage.fromJson(json.decode(String.fromCharCodes(base64.decode(data))) as Map<String, dynamic>);
     log.fine(buildPushMessage.userData);
+    log.fine('Updating buildId=${buildPushMessage.build?.id} for builder=${buildPushMessage.build?.result}');
```

Comment:
does .result contain the builder name?

Reply:
No, it's an enum of cancelled, success, infra_failure, failure

Reply:
Then change `...for builder=` to something like `...with result=`?

Reply:
Ah, thanks! Done